ShippingZones.tsx

```diff
@@ -79,18 +77,20 @@ const ShippingZonesList = () => {
     setLoading(true);
 
     try {
+      sendAnalyticsEvent("shipping_zones_list_loaded");
       const zones = await fetchShippingZones();
       setData(zones);
-      sendAnalyticsEvent("shipping_zones_shown");
     } catch (error) {
       console.log(error);
+      sendAnalyticsEvent("shipping_zones_fetch_failed");
       showRetryAlert();
     }
 
     setLoading(false);
   };
 
   useEffect(() => {
+    sendAnalyticsEvent("shipping_zones_view_shown");
```

Comment:
Usually, on Android, we have event view_shown with property name. But for now, we can't send additional properties through the native module. To save us from updating Android and iOS native modules, I think this event is fine for now.